src/System.Collections/tests/System.Collections.Tests.csproj

```diff
@@ -3,6 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <PropertyGroup>
     <ProjectGuid>{F5EB9630-AD29-4880-963F-F2D39C684D8A}</ProjectGuid>
+    <DefineConstants Condition="'$(TargetGroup)'=='netcoreapp'">$(DefineConstants);netcoreapp</DefineConstants>
```

Comment:
I'm updating this to delete that DefineConstants line since I will be moving the netcoreapp specific tests to a new file. Leave the netcoreapp build configuration and leave netstandard only?